comparisons/comparison-bpmn.md

```diff
@@ -0,0 +1,454 @@
+# Comparisons - BPMN2
+
+The [Business Process Model and Notation (BPMN)](https://www.omg.org/spec/BPMN/2.0/PDF) defines a flowchart-based
+DSL for workflows. It is maintained by the [Object Management Group (OMG)](https://www.omg.org/). 
+The latest version of BPMN is [2.0.2](https://www.omg.org/spec/BPMN/2.0.2/), published in 2014.
+
+BPMN2 defines a graphical notation to specify workflows. This notation can then be shared between tooling and organizations.
+The graphical notation is translated into XML which then can be used for execution on runtime engines.
+
+For this comparison we will compare the Serverless Workflow language with the graphical representation of BPMN2,
+and not its underlying XML representation. The BPMN2 XML is very difficult to understand, quite large for even the smallest 
+workflows, and often times not portable between runtimes. 
+It makes more sense to use its portable graphical notation for comparisons.
+
+Serverless Workflow is a declarative workflow language, represented with JSON or YAML. It currently does not 
+define a graphical notation however can be graphically represented using different flowcharting techniques such as 
+UML activity diagrams. The [Serverless Workflow Java SDK](https://github.com/serverlessworkflow/sdk-java#building-workflow-diagram) 
+as well as its [VSCode Extension](https://github.com/serverlessworkflow/vscode-extension) provide means to generate SVG diagrams based on the workflow
+JSON/YAML. 
```

Comment:
```suggestion
Serverless Workflow is a declarative workflow language, represented with JSON or YAML. It currently does not define a graphical notation. However, it can be graphically represented using different flowcharting techniques such as 
UML activity diagrams. The [Serverless Workflow Java SDK](https://github.com/serverlessworkflow/sdk-java#building-workflow-diagram) as well as its [VSCode Extension](https://github.com/serverlessworkflow/vscode-extension) provide means to generate SVG diagrams based on the workflow JSON/YAML. 
```